### PR TITLE
Update CA bundle

### PR DESCRIPTION
**Pull request by LukasReschke**
Sync CA bundle with upstream.

cc @MorrisJobke @DeepDiver1975

### CODE REVIEW

File: config/ca-bundle.crt [hunk 1]
```diff
@@ -2277,28 +2233,6 @@ yULyMtd6YebS2z3PyKnJm9zbWETXbzivf3jTo60adbocwTZ8jx5tHMN1Rq41Bab2XD0h7lbwyYIi
 LXpUq3DDfSJlgnCW
 -----END CERTIFICATE-----
 
-E-Guven Kok Elektronik Sertifika Hizmet Saglayicisi
```
Comment (LukasReschke) [inline config/ca-bundle.crt]: Removed as per https://groups.google.com/forum/#!msg/mozilla.dev.security.policy/LKJO9W5dkSY/9VjSJhRfraIJ

File: config/ca-bundle.crt [hunk 2]
```diff
@@ -1633,33 +1616,6 @@ JOzHdiEoZa5X6AeIdUpWoNIFOqTmjZKILPPy4cHGYdtBxceb9w4aUUXCYWvcZCcXjFq32nQozZfk
 vQ==
 -----END CERTIFICATE-----
 
-TC TrustCenter Class 3 CA II
```
Comment (LukasReschke) [inline config/ca-bundle.crt]: Removed as per https://bugzilla.mozilla.org/show_bug.cgi?id=1105374

File: config/ca-bundle.crt [hunk 3]
```diff
@@ -3857,4 +3791,198 @@ h5SHDDqFSmafnVmTTZdhBoZKo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUwAwEB/zAd
 BgNVHQ4EFgQUPeYpSJvqB8ohREom3m7e0oPQn1kwCgYIKoZIzj0EAwMDaAAwZQIxAOVpEslu28Yx
 uglB4Zf4+/2a4n0Sye18ZNPLBSWLVtmg515dTguDnFt2KaAJJiFqYgIwcdK1j1zqO+F4CYWodZI7
 yFz9SO8NdCKoCOJuxUnOxwy8p2Fp8fc74SrL+SvzZpA3
+-----END CERTIFICATE-----
+
+Staat der Nederlanden Root CA - G3
```
Comment (LukasReschke) [inline config/ca-bundle.crt]: As per https://bugzilla.mozilla.org/show_bug.cgi?id=1016568

File: config/ca-bundle.crt [hunk 4]
```diff
@@ -3857,4 +3791,198 @@ h5SHDDqFSmafnVmTTZdhBoZKo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUwAwEB/zAd
...
+5SWEXwbKwYY5EdtYzXc7LMJMD16a4/CrPmEbUCTCwPTxGfARKbalGAKb12NMcIxHowNDXLldRqAN
+b/9Zjr7dn3LDWyvfjFvO5QxGbJKyCqNMVEIYFRIYvdr8unRu/8G2oGTYqV9Vrp9canaW2HNnh/tN
+f1zuacpzEPuKqf2evTY4SUmH9A4U8OmHuD+nT3pajnnUk+S7aFKErGzp85hwVXIy+TSrK0m1zSBi
+5Dp6Z2Orltxtrpfs/J92VoguZs9btsmksNcFuuEnL5O7Jiqik7Ab846+HUCjuTaPPoIaGl6I6lD4
+WeKDRikL40Rc4ZW2aZCaFG+XroHPaO+Zmr615+F/+PoTRxZMzG0IQOeLeG9QgkRQP2YGiqtDhFZK
+DyAthg710tvSeopLzaXoTvFeJiUBWSOgftL2fiFX1ye8FVdMpEbB4IMeDExNH08GGeL5qPQ6gqGy
+eUN51q1veieQA6TqJIc/2b3Z6fJfUEkc7uzXLg==
+-----END CERTIFICATE-----
+
+IdenTrust Commercial Root CA 1
```
Comment (LukasReschke) [inline config/ca-bundle.crt]: As per https://bugzilla.mozilla.org/show_bug.cgi?id=1037590

File: config/ca-bundle.crt [hunk 5]
```diff
@@ -3857,4 +3791,198 @@ h5SHDDqFSmafnVmTTZdhBoZKo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUwAwEB/zAd
...
+dXNlIG9ubHkxMzAxBgNVBAMTKkVudHJ1c3QgUm9vdCBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eSAt
+IEVDMTB2MBAGByqGSM49AgEGBSuBBAAiA2IABIQTydC6bUF74mzQ61VfZgIaJPRbiWlH47jCffHy
+AsWfoPZb1YsGGYZPUxBtByQnoaD41UcZYUx9ypMn6nQM72+WCf5j7HBdNq1nd67JnXxVRDqiY1Ef
+9eNi1KlHBz7MIKNCMEAwDgYDVR0PAQH/BAQDAgEGMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYE
+FLdj5xrdjekIplWDpOBqUEFlEUJJMAoGCCqGSM49BAMDA2cAMGQCMGF52OVCR98crlOZF7ZvHH3h
+vxGU0QOIdeSNiaSKd0bebWHvAvX7td/M/k7//qnmpwIwW5nXhTcGtXsI/esni0qU+eH6p44mCOh8
+kmhtc9hvJqwhAriZtyZBWyVgrtBIGu4G
+-----END CERTIFICATE-----
+
+CFCA EV ROOT
```
Comment (LukasReschke) [inline config/ca-bundle.crt]: As per https://bugzilla.mozilla.org/show_bug.cgi?id=926029

**Comment by LukasReschke** [comment]
Manually verified. LGTM from my side.

**Comment by DeepDiver1975** [comment]
we need that on stable8 as well?

**Comment by LukasReschke** [comment]
> we need that on stable8 as well?

Just master. – That cert file is part of my very big huge 8.1 change workaround by including our own certificate bundle… Which basically means somebody (=== me) needs to sync our certificate every few months with upstream :see_no_evil: 

Still better than all those people complaining about broken HTTPS etc…

**Comment by MorrisJobke** [comment]
> Just master. – That cert file is part of my very big huge 8.1 workaround by including our own certificate bundle… Which basically means somebody (=== me) needs to sync our certificate bundle every few months with upstream :see_no_evil:
> 
> Still better than all those people complaining about broken HTTPS etc…

*cough* cron job *cough*

**Comment by MorrisJobke** [comment]
Looks good :+1:

**Comment by LukasReschke** [comment]
> cough cron job cough

*cough* we will not have an cronjob automatically do security stuff *cough* ;-)

**Comment by MorrisJobke** [comment]
> cough we will not have an cronjob automatically do security stuff cough ;-)

*cough* But we could have a cronjob that will ping you once there is a change *cough*

**Comment by LukasReschke** [comment]
> cough But we could have a cronjob that will ping you once there is a change cough

*cough* sooooo true *cough*

**Comment by PVince81** [comment]
Makes sense :+1:

**Pull request merged**